worker/src/files.js

```diff
@@ -0,0 +1,50 @@
+const axios = require('axios');
+const FormData = require('form-data');
+const https = require('https');
+const { downloadFile } = require('./s3');
+const { File } = require('./models');
+
+const fileStatuses = {
+  uploading: 'UPLOADING',
```

Comment:
Looks like these are being used as enums? I believe the naming conventions for the values is to all be UPPER_CAMEL_CASE